common/util.c

```diff
@@ -71,6 +71,16 @@ float parse_float(const char *value) {
 	return flt;
 }
 
+bool parse_unit(const char *str, bool *ppt) {
+	if (strcmp(str, "px") == 0) {
+		if (ppt) *ppt = false;
```

Comment:
Style: braces are mandatory

But I don't think we need this check, we never call the function with a NULL `ppt`.